crates/polars-core/src/series/implementations/null.rs

```diff
@@ -43,7 +43,19 @@ impl PrivateSeriesNumeric for NullChunked {}
 
 impl PrivateSeries for NullChunked {
     fn compute_len(&mut self) {
-        // no-op
+        // NullChunked only has one chunk.
+        self.length = self.chunks[0].len() as IdxSize;
```

Comment:
`amortized_iter_with_name` will change the chunk directly during iteration, but the length will not be re-compute before.

Reply:
I think we should sum the length of all chunks. We also should convert via `IdxSize::from` and remove the asserts below (they are wrong). See the `compute_len` on `ChunkedArray<T>`.

Reply:
> I think we should sum the length of all chunks.

Ah, I'm not realize that we had already made null chunking behavior equal to other dtypes before.

> We also should convert via IdxSize::from and remove the asserts below (they are wrong).

Make sense, seems that we just fixed this a few days ago for `ChunkedArray<T>`.

Updated this.